app/src/main/java/ai/elimu/analytics/entity/NumberLearningEvent.kt

```diff
@@ -0,0 +1,14 @@
+package ai.elimu.analytics.entity
+
+import ai.elimu.model.v2.enums.analytics.LearningEventType
+import androidx.room.Entity
+
+@Entity
+class NumberLearningEvent : LearningEvent() {
+
+    var numberId: Long? = null
```

Comment:
@tuancoltech We should also include `numberValue` here.

Reply:
@tuanctech The entity in the Analytics DB should reflect the entity in the backend DB: https://github.com/elimu-ai/webapp/blob/main/src/main/java/ai/elimu/entity/analytics/NumberLearningEvent.java

Some languages have their own symbol for numbers: http://hin.elimu.ai/content/number/list